crm_track_next_action/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM Track Next Action
+=====================
+
+This module adds tracking when the next action fields are changed on an opportunity.
```

Comment:
Please explain that it will appear messages on the chatter, because users don't really understand what "tracking" means.